platforms/core-configuration/configuration-problems-base/src/main/kotlin/org/gradle/internal/cc/impl/problems/JsonModelWriter.kt

```diff
@@ -55,49 +49,39 @@ class JsonModelWriter(val writer: Writer) {
 
     fun endModel(details: ProblemReportDetails) = with(details) {
         endArray()
-
         comma()
```

Comment:
❓  Is this comma required here?

Reply:
yes, it breaks otherwise

Reply:
This is rather strange, because the following `property()` call should insert a comma automatically, given that we call `endArray()` and decrease a level.

Should we add some tests for the Json writing utility?